test/remote/password_forgot_tests.js

```diff
@@ -31,6 +31,7 @@ TestServer.start(config)
         .then(
           function (x) {
             client = x
+            client.setAllowedOrigin(config.corsOrigin)
```

Comment:
Yeah, fair point. Will come back and sort this out properly when my content server PR is done.